.travis.yml

```diff
@@ -9,14 +9,14 @@ branches:
 
 env:
   # default is stable repo
-  - DISTRO=ubuntu-16
   - DISTRO=ubuntu-18
+  - DISTRO=ubuntu-20
```

Comment:
Need to remove ubuntu-20 from here. We can't run it as a stable run yet until 3.5 is released. 
So we need to do the changes for ansible-st2 in 2 stages.
1) add it to unstable before release - under this PR
2) add it to stable after release - separate PR

Reply:
I think we'll just have to wait for a release to happen before merging this PR.

Otherwise, someone using the fresh Ansible installer will have the same issues with U16/U20.